### ISSUE TITLE
Inaccessibility: Low contrast text color in dark mode

### ISSUE THREAD

**Issue by ruthmoog** [a11yproject/a11yproject.com#1613]
In dark mode, some text is too low contrast. eg "tl;dr", "&c" is in #383838 on background #222222 which is only a contrast ratio of 1.4:1.

The contrast ratio should be at least 3:1 (WCAG 2.1 for scaled text).

**Comment by SaptakS** [comment]
@ruthmoog can you share screenshots or which links you are seeing the contrast issues?

**Comment by ericwbailey** [comment]
These are decorative images and exempt.
